pkg/operator/config/manager/kustomization.yaml

```diff
@@ -11,4 +11,4 @@ kind: Kustomization
 images:
 - name: controller
   newName: thundernetes-operator
-  newTag: 495b519
```

Comment:
Looks like I overwrote this in my last pr, can you change it back to the original values?
newName: ghcr.io/playfab/thundernetes-operator
newTag: 0.3.2

Reply:
nice catch, thanks. We should file an issue to somehow fix this